### DISCUSSION TITLE
How do I load a game into my plugin?

### DISCUSSION THREAD

**Discussion by vismodo** [ScreamingSandals/BedWars#378] [Q&A] [answered]
I put this question as an issue already, but here it is again.

I'm new to Java and Spigot, so bear with me. I'm trying to make my plugin automatically connect a player to a game. I found that that can be done using `joinToGame(player)` on a game instance. I want to know how to retrieve the correct game instance using the arena's name. I don't know if I have phrased this properly, so I'm going to put what I want to do, in a line of code:

```java
arena.joinToGame(player);
```

Thanks!

**Comment by zlataovce** [comment, marked as answer]
check out the javadocs at https://jd.screamingsandals.org/sbw-0-2-x/BedWars-API/
